selfdrive/test/process_replay/test_processes.py

```diff
@@ -154,36 +181,43 @@ def format_diff(results, ref_commit):
     assert len(untested) == 0, f"Cars missing routes: {str(untested)}"
 
   results: Any = {}
+  pool_args: Any = []
   for car_brand, segment in segments:
     if (len(args.whitelist_cars) and car_brand.upper() not in args.whitelist_cars) or \
        (not len(args.whitelist_cars) and car_brand.upper() in args.blacklist_cars):
       continue
-
-    print(f"***** testing route segment {segment} *****\n")
-
-    results[segment] = {}
-
-    r, n = segment.rsplit("--", 1)
-    lr = LogReader(get_url(r, n))
-
+    if not args.parallel:
+      print(f"***** testing route segment {segment} *****\n")
+      results[segment] = {}
+      r, n = segment.rsplit("--", 1)
+      lr = LogReader(get_url(r, n))
```

Comment:
this shouldn't be a completely separate code path. process replay should just take something like a jobs argument

Reply:
I did it that way initially, but there is then a lot of duplicate code. I will change it back and we can see